src/transforms/samplesTransforms.ts

```diff
@@ -117,22 +122,31 @@ export async function getAllExamples(codeModel: TestCodeModel, clientDetails: Cl
             sample.clientParameterNames = clientParameterNames.join(", ");
           }
           const methodParameterNames = [];
-          const optionalParams = [];
+          const optionalParams: [string, string][]= [];
           for (const methodParameter of example.methodParameters) {
             if (
               methodParameter.exampleValue.schema.type === SchemaType.Constant
             ) {
               continue;
             }
-            const parameterName = getLanguageMetadata(
-              methodParameter.exampleValue.language
-            ).name;
+            const parameterName = normalizeName(
+              getLanguageMetadata(methodParameter.exampleValue.language).name,
+              NameType.Parameter,
+              true
+            );
+            const parameterTypeDetails = getTypeForSchema(
+              methodParameter.exampleValue.schema
+            );
+            const parameterTypeName = parameterTypeDetails.typeName;
             let paramAssignment = "";
             if (methodParameter.parameter.protocol?.http?.["in"] === "body") {
               sample.hasBody = true;
-              sample.bodySchemaName = getLanguageMetadata(
-                methodParameter.exampleValue.schema.language
-              ).name;
+              sample.bodySchemaName = parameterTypeName;
+              sample.importedTypes?.push(parameterTypeName);
+              if (methodParameter.exampleValue.schema.type === SchemaType.AnyObject || methodParameter.exampleValue.schema.type  === SchemaType.Any) {
+                sample.bodySchemaName = "Record<string, unknown>"
+                sample.isAnyTypeBody = true;
```

Comment:
We re-correct the value of bodySchemaName & isAnyTypeBody but we didn't put the latest into importedTypes list or any usage directly, do we need to refine this?